ipaclient/install/client.py

```diff
@@ -1157,14 +1170,22 @@ def modify_ssh_config(options):
 
 def create_ssh_ipa_config(options):
     """Add the IPA snippet for ssh"""
+    # sss_ssh_knownhostsproxy is deprecated in favor of sss_ssh_knownhosts
+    # use sss_ssh_knownhosts when possible
+    enableknownhosts = bool(
+        options.sssd and os.path.isfile(paths.SSS_SSH_KNOWNHOSTS)
+    )
+
     enableproxy = bool(
-        options.sssd and os.path.isfile(paths.SSS_SSH_KNOWNHOSTSPROXY)
+        options.sssd and os.path.isfile(paths.SSS_SSH_KNOWNHOSTSPROXY) and not enableknownhosts  # noqa: E501
```

Comment:
Please add a line break instead of ignoring the linter error

Reply:
fixed